Lib/gftools/scripts/rename_glyphs.py

```diff
@@ -0,0 +1,30 @@
+"""
+Rename glyph names in font_1 with font_2.
+
+Usage:
+gftools rename-glyphs font_1.ttf font_2.ttf out font_1-renamed.ttf
+"""
+from fontTools.ttLib import TTFont
+from ufo2ft.postProcessor import PostProcessor, _reloadFont
+from argparse import ArgumentParser
+
+
+def main(args=None):
+    parser = ArgumentParser(description=__doc__)
+    parser.add_argument("font_1")
+    parser.add_argument("font_2")
+    parser.add_argument("-o", "--out", default="out-renamed.ttf")
+    args = parser.parse_args(args)
+
+    font1 = TTFont(args.font_1)
+    font2 = TTFont(args.font_2)
+    
+    name_map = dict(zip(font1.getGlyphOrder(), font2.getGlyphOrder()))
+
+    font1 = _reloadFont(font1)
```

Comment:
Is this really needed? It makes sense in the context of ufo2ft, but I'm not sure about here. Although I can see perhaps there's a need to ensure the font is fully decompiled before you start messing with glyph names (since everything in fontTools is referenced by name, not ID).

Either remove if it's not needed, or add a comment about why it is, because currently it's not obvious. (I just loaded the font and now I have to reload it?)